### PR TITLE
Fixes #98: Use ripple instead of bold text to indicate focus.

### LINKED ISSUE #98: font weighting issue throwing positioning off
maybe im just being nitpicky here, but when a tab is focused, its font weight is changed to 700. this throws off the width of the tab bar and from an end user perspective looks unpolished.

### PR DESCRIPTION
**Pull request by bicknellr**
screen shot 2016-04-04 at 18 09 58

### CODE REVIEW

**Comment by notwaldorf** [comment]
Let's double check with @zgibson if this what he envisioned for the focused state

**Comment by notwaldorf** [comment]
@bicknellr Did you get a chance to talk to Zach about this?

**Comment by bicknellr** [comment]
Not yet. ping @zgibson

**Comment by notwaldorf** [comment]
@bicknellr I don't think he checks github, so you might have to talk to him the old fashioned way

**Comment by bicknellr** [comment]
(recap from hangouts discussion) Zach confirmed that this is actually the intended focus state for tabs.

File: paper-tab.html [hunk 1]
```diff
@@ -169,8 +172,34 @@
 
           anchor.click();
         }
-      }
+      },
 
+      _focusedChanged: function(receivedFocusFromKeyboard) {
+        var ripple = this.getRipple();
+
+        if (this.focused && receivedFocusFromKeyboard) {
```
Comment (notwaldorf) [inline paper-tab.html]: Can you do something simpler like what we do for the InkyFocusBehavior? https://github.com/PolymerElements/paper-behaviors/blob/master/paper-inky-focus-behavior.html#L26

File: paper-tab.html [hunk 2]
```diff
@@ -169,8 +172,34 @@
...
+          }
+          ripple.downAction();
+        }
+
+        if (ripple) {
+          ripple.holdDown = receivedFocusFromKeyboard;
+        }
+      },
+
+      _asyncClick: function() {
```
Comment (notwaldorf) [inline paper-tab.html]: Is this used?

Comment (bicknellr) [inline paper-tab.html]: It's overriding a method on IronButtonState.

File: paper-tab.html (hunk 1 above)
Comment (bicknellr) [inline paper-tab.html]: Switched this to use PaperInkyFocusBehavior directly.

File: paper-tab.html [hunk 3]
```diff
@@ -169,8 +164,25 @@
...
+      },
+
+      _createRipple: function() {
+        var ripple = Polymer.PaperInkyFocusBehaviorImpl._createRipple();
+        ripple.removeAttribute('center');
+        ripple.classList.remove('circle');
+        return ripple;
+      },
 
+      _asyncClick: function() {
```
Comment (notwaldorf) [inline paper-tab.html]: I vaguely remember commenting on this, but can't see where the comment went -- why is this needed? Can you at least add a comment why it's needed, because it doesn't seem to be called anywhere in this file.

Comment (bicknellr) [inline paper-tab.html]: Yeah, it was collapsed after I pushed something over it and it decided this was a different line. This is overriding a function in IronButtonState.

Comment (notwaldorf) [inline paper-tab.html]: Sure, but can you please document that? And most importantly, document how it's different, or why the original implementation doesn't work

Comment (bicknellr) [inline paper-tab.html]: Ok, will do.

**Comment by abdonrd** [comment]
Any news?
